src/dists.ad.js

```diff
@@ -717,6 +751,59 @@ var TensorGaussian = makeDistributionType({
   }
 });
 
+function tensorLaplaceSample(mu, b, dims) {
+  var x = new Tensor(dims);
+  var n = x.length;
+  while (n--) {
+    x.data[n] = laplaceSample(mu, b);
+  }
+  return x;
+}
+
+function tensorLaplaceScore(mu, b, dims, x) {
+  var _x = ad.value(x);
+  var scOp = ad.scalar;
+  var tnOp = ad.tensor;
```

Comment:
* You could use the ad transform here, then use `*/+-` for the scalar ops. This will also fix what looks like a bug in the current implementation, that arises from doing `2*b` when `b` is an ad graph node.
* `T` is possible already bound to `ad.tensor` here.